openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1224,7 +1226,7 @@ Error codes extensions defined in (#credential-error-response) apply.
 
 The Batch Credential Request MUST fail entirely if there is even one Credential that failed to be issued. `transaction_id` MUST NOT be returned in this case.
 
-When the Credential Issuer requires the `proofs` parameter to be present in the Batch Credential Request, but does not receive them, it will return a Batch Credential Error Response with a `c_nonce` using `invalid_proof` error code, as defined in (#issuer-provided-nonce).
+When the Credential Issuer requires the `proofs` parameter to be present in the Batch Credential Request, but does not receive them, it will return a Batch Credential Error Response using `invalid_proof` error code, as defined in (#issuer-provided-nonce).
```

Comment:
umm, we still should keep an option to return c_nonce in error, no?

Reply:
I removed this language because I fixed the "Credential Issuer Provided Nonce" section to also include Batch Credential Requests/Responses. There is no need for keeping this section here. Does this make sense?

Reply:
I actually think, the entire sentence should probably get removed.

Reply:
This entire section will be deleted by https://github.com/openid/OpenID4VCI/pull/364 so (on the assumption that PR will be merged) it's best not to change text in the batch credential endpoint section in this PR.

Reply:
I actually think, we should make this consistent and remove the entire sentence. I know there might be merge conflicts, but let's see which PR we can merge first. If this PR gets merged first, then I don't feel comfortable still having inconsistent language in the spec on `c_nonce`.

Reply:
batch credential endpoint PR has been merged, please adapt the text accordingly :)